### PR TITLE
COMP: Should not complete for function with test attribute

### LINKED ISSUE #1189: Autocomplete currently suggests test functions
Using Hyper for example the functions defined with #[test] are suggested in the autocomplete. If its possible to exclude any #[test] method from autocompleting that would be superb and would let us more easily see the methods we are interested in.

Btw, keep up the good work - you're doing brilliantly!

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1189

### CODE REVIEW

File: src/main/kotlin/org/rust/lang/core/resolve/Processors.kt [hunk 1]
```diff
@@ -52,6 +55,8 @@ fun collectResolveVariants(referenceName: String, f: (RsResolveProcessor) -> Uni
 fun collectCompletionVariants(f: (RsResolveProcessor) -> Unit): Array<LookupElement> {
     val result = mutableListOf<LookupElement>()
     f { e ->
+        if((e.element as? RsFunction?)?.isTest ?: false)
```
Comment (matklad) [inline src/main/kotlin/org/rust/lang/core/resolve/Processors.kt]: A space is missing after the `if` ;) 

Also we usually format short ifs on a single line: `if (cond) return false`, or add `{}`.

Comment (farodin91) [inline src/main/kotlin/org/rust/lang/core/resolve/Processors.kt]: Is this the correct position?

Comment (matklad) [inline src/main/kotlin/org/rust/lang/core/resolve/Processors.kt]: yep!

**Comment by matklad** [comment]
Thanks!

bors r+

**Pull request merged**